packages/chakra-components/src/MenuBar/SubMenu.js

```diff
@@ -34,7 +34,6 @@ const SubMenu = ({
 
   const _isOpen = isControlled ? isOpenProp : isOpen;
   const wasPreviouslyOpen = usePrevious(_isOpen);
-  const wasPreviouslyOpenBeforeTimeout = usePrevious(wasPreviouslyOpen);
```

Comment:
Why was this removed?

Reply:
rebase with refractor-menubar and it's not being used in this file.

Reply:
Done 👍